index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="author" content="Emmanuel Hagan" />
+  <title>Hagan's Portifio</title>
+    <!-- Stylesheet -->
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+
+  <!--  Header  -->
+  <header class="toolbar-container">
+      <a href="#" class="logo-heading" > Hagan</a>
+   <!---  
+      <nav class="hamburger-menu">
+       <ul class="header-nav-links">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li> 
+        </ul>
+      </nav>
+    -->
+      <img class="icon" src="assets/image/icon-menu.png"  alt="Icon menu">
+  </header>
+  <!-- End Header  -->
+
+  <!-- Section  -->
+  <section class="headline">
+    <div class="headline-content-wrapper">
+      <h1 class="intro">I’m Emmanuel Hagan Glad to see you!</h1>
+    </div>
+      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      
+        <a href="" class="connect">
+          <h1>LET’S CONNECT</h1>
+        </a>
+      
+      <!---Social Link-->
+      <nav class="social-links">
+        <ul>
+          <li>
+            <a href=""><img src="assets/image/social_icon/medium.png"  alt="Medium"></a>
+          </li>
+          <li>
+            <a href=""><img src="assets/image/social_icon/angellist.png"  alt="angellist"></a>
+          </li>
+          <li>
+            <a href=""><img src="assets/image/social_icon/twitter.png"  alt="twitter"></a>
+          </li>
+          <li>
+            <a href=""><img src="assets/image/social_icon/github.png"  alt="github"></a>>
+          </li>
+
+
+          <!---End Social Link-->
+        </ul>
```

Comment:
- [ ] Please make sure to have the "LET'S CONNECT" text aligned on the left side with the list of icons.

Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/164814906-966a3cdf-e541-4314-8092-748e3f504e4a.png)|![image](https://user-images.githubusercontent.com/18730106/164814946-14785450-c86a-44b6-8db1-8c74347c7d3c.png)